src/common/options/global.yaml.in

```diff
@@ -3228,7 +3228,7 @@ options:
   default: true
   with_legacy: true
 # The number of shards for holding the ops
-- name: osd_num_op_tracker_shard
+- name: num_op_tracker_shard
```

Comment:
What about already existing clusters? Will presence of the old option rise a warning, an error or it's just going to be silently ignored?

Reply:
Silently ignored, from my point of view.